File: market-application/src/main/java/com/example/market/application/command/SellNowCommand.java
package com.example.market.application.command;

import com.example.market.domain.catalog.SkuId;
import com.example.market.domain.shared.UserId;

public record SellNowCommand(
        String idempotencyKey,
        UserId sellerId,
        SkuId skuId
) {}

File: market-application/src/main/kotlin/com/example/market/application/command/SellNowCommand.kt
package com.example.market.application.command

import com.example.market.domain.catalog.SkuId
import com.example.market.domain.shared.UserId

@JvmRecord
data class SellNowCommand(
    val idempotencyKey: String,
    val sellerId: UserId,
    val skuId: SkuId,
)
